test/releasable-test.js

```diff
@@ -0,0 +1,211 @@
+'use strict';
+
+const { describe, it } = require('./helpers/mocha');
+const { expect } = require('./helpers/chai');
+const { getChangedReleasableFiles } = require('../src/releasable');
+const fixturify = require('fixturify');
+const stringifyJson = require('../src/json').stringify;
+const { gitInit } = require('git-fixtures');
+const path = require('path');
+
+describe(function() {
+  let tmpPath;
+
+  beforeEach(async function() {
+    tmpPath = await gitInit();
+  });
+
+  describe(getChangedReleasableFiles, function() {
+    it('works with files array', async function() {
+      fixturify.writeSync(tmpPath, {
```

Comment:
why writeSync when we're in an async function?

Reply:
That's all that library offers.